### DISCUSSION TITLE
[atum]Remove sidebar menu on media manager

### DISCUSSION THREAD

**Discussion by micker** [joomla/joomla-cms#33619] [2. Cold Features]
hello
what do you think about remove sidebar menu on media manager
- more coherent than other view for editing
- more simplier for accessibility less link
- more space for image
Before
![image](https://user-images.githubusercontent.com/2539440/117413140-e371eb80-af15-11eb-87d9-2e7419767d0c.png)

After
![image](https://user-images.githubusercontent.com/2539440/117413217-fc7a9c80-af15-11eb-853b-ab0dd80c7087.png)

**Comment by leoalv** [comment]
For that there is TOGGLE MENU, it is below the logo.

**Comment by dgrammatiko** [comment]
So, how exactly can you navigate from the media manager to another view if there is no menu?

**Reply by micker** [reply]
for me its like article editing no menu, if we want to go out we can use joomla logo ?
in my first screen we can see that menu + folder tree reduce the srcreen usuable not ?

**Comment by dgrammatiko** [comment]
> for me its like article editing no menu, if we want to go out we can use joomla logo ?

If the logo is taking you out of an edit form that's a bug, it shouldn't. The only way to exit it to either save & exit or cancel...

Edit: Just tested that and it doesn't work as expected

**Comment by leoalv** [comment]
I'll repeat it again, use the toggler
![screenshot-localhost-2021 05 09-14_15_18](https://user-images.githubusercontent.com/5317182/117584179-10b6c900-b0d1-11eb-9409-7b06bdd862a5.png)
